application/classes/Ushahidi/Formatter/Post/GeoJSONCollection.php

```diff
@@ -47,6 +47,20 @@ public function __invoke($entities)
 
 			if (! empty($geometries))
 			{
+				$set_name = [];
+				$set_ids = $entity->sets;
+
+				if ($set_ids != null) {
+					$query = DB::select('sets.name')
```

Comment:
Also - we'd normally only include the `id` for relations, not the name as well. Definitely not only the name.
Could you make your client changes work using just the collection `id`?